test/with_api_v1/acceptance/send_test.go

```diff
@@ -272,10 +271,7 @@ receivers:
 func TestResolved(t *testing.T) {
 	t.Parallel()
 
-	var wg sync.WaitGroup
-	wg.Add(10)
-
-	for i := 0; i < 10; i++ {
+	for i := 0; i < 2; i++ {
 		go func() {
```

Comment:
Does go testing wait until all references to `t *testing.T` are gone? Otherwise this test will finish before all go routines are done, right?

Reply:
Oops, I forgot to remove the `go func() {}`.